src/cr/cube/cubepart.py

```diff
@@ -291,6 +293,13 @@ def columns_dimension_type(self):
     def columns_margin(self):
         return np.array([column.margin for column in self._matrix.columns]).T
 
+    @lazyproperty
+    def columns_moe(self):
+        """Returns the margin of error (MoE) for cell percentages
+        `moe = z(0.25) * 100 * std_error`
```

Comment:
i don't understand the `z(0.25)` and even why are you multiplying by 100. can you give me more context?